R/check-cran.sh

```diff
@@ -40,7 +40,7 @@ fi
 
 if [ -d "$SPARK_JARS_DIR" ]; then
   # Build a zip file containing the source package with vignettes
-  SPARK_HOME="${SPARK_HOME}" "$R_SCRIPT_PATH/"R CMD build $FWDIR/pkg
+  SPARK_HOME="${SPARK_HOME}" "$R_SCRIPT_PATH/"R CMD build "$FWDIR"/pkg
 
```

Comment:
Sure, but i do not think it is a good choice to quote whole arguments that include a variable.
e.g.  cp "/home/spark build/spark/conf/*.template" "/home/test"  will failed.
It seems to me that quote the variable is better than  whole arguments, what is your suggestion?
Thanks!